autossh/run.sh

```diff
@@ -46,6 +51,15 @@ if [ -z "$HOSTNAME" ]; then
   exit 1
 fi
 
+local_forward_uri="$FORWARD_LOCAL_IP_ADDRESS:$FORWARD_LOCAL_PORT"
+bashio::log.info "Checking accessibility of $local_forward_uri"
+status_code=$(curl --write-out %{http_code} --silent --output /dev/null $local_forward_uri)
+if [[ "$status_code" -ne 200 ]] ; then
+  bashio::log.error "Check failed. $local_forward_uri resulted HTTP status_code=$status_code and not 200. Is the address correct?" && exit 1
+else
+  bashio::log.info "Check passed. $local_forward_uri returned HTTP status_code=200"
```

Comment:
I would suggest a less technical message like 
```suggestion
  bashio::log.info "Testing local port definition... Home Assistant web frontend can be reached"
```

Reply:
Done, good thinking. I kept the error one a bit more technical to provide some additional debug info e.g. in the cases people of some forwarding turned on (301 will now fail).